src/validation/userValidation.js

```diff
@@ -1,19 +1,26 @@
 import { check } from 'express-validator';
 
-const userValidation = [
+const userProfileValidation = [
   check('firstName').trim().not().isEmpty()
     .withMessage('First name field cannot be empty.'),
   check('lastName').trim().not().isEmpty()
     .withMessage('last name field cannot be empty.'),
-  check('email').trim().not().isEmpty()
-    .withMessage('Email field cannot be empty'),
-  check('email').isEmail().withMessage('Enter valid email address.'),
-  check('email').normalizeEmail(),
   check('password').trim().not().isEmpty()
     .withMessage('Please password is required'),
   check('password').isLength({ min: 8 }).withMessage('Password should be atleast 8 characters'),
```

Comment:
The user will not be changing their password on their profile. There should be a reset password feature for that

Reply:
Done